app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -71,7 +71,15 @@ keywords:
 
 $new-international-funding$
 
-Citizens of every country in the world can apply to train to teach in England.
+Citizens of every country in the world can apply to train to teach in England. When you plan your training, bear in mind that:
+
+- international candidates can expect to pay an average of £14,765, plus
+living costs, to train
```

Comment:
@laurahermionetennant92 the 'to train' might be better after the amount so it reads: international candidates can expect to pay an average of £14,765 to train, plus living costs